packages/drag-drop/demo/~patterns/patterns.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
+import { SortableListsStory } from './stories/SortableListsStory';
+import { COLUMNS as columns } from './stories/data';
+
+<Meta title="Packages/Drag-Drop/[patterns]" />
+
+# Patterns
+
+## Sortable Lists
+
+Use `DraggableList`, `Dropzone`, and `Draggable` to create sortable lists.
+
+<Canvas>
+  <Story
+    name="Sortable Lists"
+    args={{ columns }}
+    argTypes={{
```

Comment:
I'd like to see a _lot_ more control added to this story, if possible. For example:
- ability to toggle `DropIndicator` usage
- ability to toggle placeholder usage
- toggling compact layout
- toggle danger styling
- ability to disable the components

It's going to be important to have pattern flexibility to see these changes/combos with a dynamic story. I'm fine if we work on the design of this story as a follow-on to the PR.

Reply:
Yeah, I get that. And we don't want to add hi-fi work here that is redundant with the website. But we do want to achieve a couple of targeted considerations with Storybook:
- pushing into potential (odd) combinations that the website would naturally tend to avoid
- ability to observe the affect of ongoing component maintenance in a working "playground"

Reply:
🤝 Agreed.